src/import/load-costume.js

```diff
@@ -1,7 +1,12 @@
 const StringUtil = require('../util/string-util');
 const log = require('../util/log');
+const assert = require('assert');
 
 const loadVector_ = function (costume, runtime, rotationCenter, optVersion) {
+    debugger;
```

Comment:
Is this debugger statement supposed to be here?

Reply:
Oops thanks